CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [6.1.1] - 2023-07-19
+
+### Changelog for Packages in This project
+
+Please click corresponded links for more details for changelog of each package.
+| Folder | Package | CHANGELOG |
+| ----------------------------------------------------------- | ----------------------------------------- | ------------------------------------------------------ |
+| [solutions/deployment](./solutions/deployment/) | [@aws/fhir-works-on-aws-deployment] | [Changelog](./solutions/deployment/CHANGELOG.md) |
+| [solutions/smart-deployment](./solutions/smart-deployment/) | [@aws/fhir-works-on-aws-smart-deployment] | [Changelog](./solutions/smart-deployment/CHANGELOG.md) |
+| [fwoa-core/search-es](./fwoa-core/search-es/) | [@aws/fhir-works-on-aws-search-es] | [Changelog](./fwoa-core/search-es/CHANGELOG.md) |
```

Comment:
Nit: There's no change in the search package, we can remove this line here

Reply:
Updated!